Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 <Project>
 	<PropertyGroup>
-		<Version>6.0.1.14</Version>
+		<Version>6.0.1.2</Version>
```

Comment:
Nope, revert :sunglasses:

Reply:
FFS. I definitely pressed the "revert" button in Rider when it showed me that as a merge conflict... Seems the file wasn't saved before I pressed "commit"